content/home/examples/a-simple-component.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: A Simple Component
+title: ಒಂದು ಸರಳ ಕಾಂಪೊನೆಂಟ್
 order: 0
 domid: hello-example
 ---
 
-React components implement a `render()` method that takes input data and returns what to display. This example uses an XML-like syntax called JSX. Input data that is passed into the component can be accessed by `render()` via `this.props`.
+ರಿಯಾಕ್ಟ್ ಕಾಂಪೊನೆಂಟ್ಗಳು ಇನ್ಪುಟ್ ಡೇಟಾ ತೆಗೆದುಕೊಳ್ಳುತ್ತದೆ ಮತ್ತು ಪ್ರದರ್ಶಿಸಲು ಏನು ಹಿಂದಿರುಗಿಸುತ್ತದೆ ಒಂದು `render()` ವಿಧಾನವನ್ನು ಜಾರಿಗೆ. ಈ ಉದಾಹರಣೆಯು JSX ಎಂಬ XML ಮಾದರಿಯ ಸಿಂಟ್ಯಾಕ್ಸನ್ನು ಬಳಸುತ್ತದೆ. ಘಟಕಕ್ಕೆ ವರ್ಗಾಯಿಸಲಾದ ಇನ್ಪುಟ್ ಡೇಟಾವನ್ನು `this.props`ನ ಮೂಲಕ `render()` ಮೂಲಕ ಪ್ರವೇಶಿಸಬಹುದು.
```

Comment:
1. ಕಾಂಪೊನೆಂಟ್ಗಳು should be ಕಾಂಪೊನೆಂಟ್ಸ್
2. ವಿಧಾನವನ್ನು may be replaced with ಮೆಥೋಡ್ಸ್ 
3. ಕಾಂಪೊನೆಂಟ್ಗೆ ನೀಡಲಾಗುವ ಇನ್ಪುಟ್ ಡೇಟಾ ನಾವು `this.props`ನ ಮೂಲಕ `render()` ಮೂಲಕ ಆಕ್ಸೆಸ್ ಮಾಡಬಹುದು.